dev/reference/tools/loader.md

```diff
@@ -97,13 +97,10 @@ user = "root"
 password = ""
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream will be used.
```

Comment:
```suggestion
# `sql_mode` of the session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream is used.
```

Reply:
> of the session level 

I am not sure about this one